lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/reuse-existing-identity.yaml

```diff
@@ -109,36 +96,53 @@ states:
       next:
         targetState: DETAILS_DELETED_PAGE
 
-  DETAILS_DELETED_PAGE:
+  CONFIRM_DELETE_DETAILS_PAGE:
     response:
       type: page
-      pageId: pyi-details-deleted
+      pageId: pyi-confirm-delete-details
     events:
       next:
-        targetJourney: NEW_P2_IDENTITY
-        targetState: START
+        targetState: RESET_SESSION_IDENTITY
+      end:
+        targetState: IDENTITY_REUSE_PAGE
+        checkFeatureFlag:
+          coiEnabled:
+            targetState: IDENTITY_REUSE_PAGE_COI
 
   UPDATE_DETAILS_PAGE:
     response:
       type: page
       pageId: update-details
     events:
-      address:
-        targetJourney: UPDATE_ADDRESS
-        targetState: START
-      name:
-        # TODO replace with UPDATE_NAME_ADDRESS when it is complete
-        targetJourney: UPDATE_ADDRESS
-        targetState: START
-      name-address:
-        # TODO replace with UPDATE_NAME_ADDRESS when it is complete
+      address-only:
         targetJourney: UPDATE_ADDRESS
         targetState: START
```

Comment:
I don't think this state exists - `update-address.yaml` only has entry points of `UPDATE_NAME_WITH_ADDRESS_START` and `UPDATE_NAME_WITHOUT_ADDRESS_START`.
Neither of which seem right for only updating an address interestingly.